swigibpy.py

```diff
@@ -2036,7 +2036,10 @@ def run(self):
         pollerr = [fd]
 
         while self._tws and self._tws.isConnected():
-            evts = select(pollin, pollout, pollerr)
+            try:
+                evts = select(pollin, pollout, pollerr)
+            except OSError:
```

Comment:
`OSError` will only be thrown here in Python 3. Change to  [`select.error`](https://docs.python.org/3/library/select.html) so it works in both Python 2 and 3.